hc/api/tests/test_create_check.py

```diff
@@ -93,12 +93,5 @@ def test_it_rejects_non_number_timeout(self):
     def test_it_rejects_non_string_name(self):
         self.post({"api_key": "abc", "name": False},
                   expected_error="name is not a string")
```

Comment:
any reason why  `test_it_rejects_small_timeout` and `test_it_rejects_large_timeout` were deleted?